java8/build/Dockerfile

```diff
@@ -6,8 +6,14 @@ WORKDIR /
 
 RUN rm -rf /var/runtime /var/lang && \
   curl https://lambci.s3.amazonaws.com/fs/java8.tgz | tar -zx -C / && \
-  yum install -y --releasever=latest java-1.8.0-openjdk-devel-1.8.0.181
+  yum install -y --releasever=latest java-1.8.0-openjdk-devel-1.8.0.181 && \
+  mkdir /opt/gradle && curl -L -o gradle.zip https://services.gradle.org/distributions/gradle-5.2-bin.zip && \
```

Comment:
Might be better to use `/usr/local` – or similar. As `/opt` is used by layers, people often want to use the build images to build/install things in there, and then zip everything up. Best to keep `/opt` just for layers I think.

Reply:
Ahh good point. I just followed what Gradles docs suggested. Will move to `/usr/local`.